t/docker/lib/common.sh

```diff
@@ -63,7 +63,7 @@ chown "$dbuser" /var/lib/openqa/.config/openqa/client.conf
 systemctl enable --now rsyncd
 
 if [ "$METHOD" == rest ]; then
-    openqa-client --host localhost /api/v1/obs_rsync/$prj/runs put
+    openqa-cli api -X put --host localhost /api/v1/obs_rsync/$prj/runs
```

Comment:
1. needs http:// , otherwise it will try to force https:// which is not configured in the test
2. Since apache is not set up in the test, it needs WebUI port here, so http;//localhost:9526
3. It still doesn't work - - needs an example how to put into /api/v1 (see my first comment in the thread).
I think we can just leave old openqa-client in the tests (folder t/docker) and change it with separate commit once we have confirmation how to properly use it